src/main/java/ru/bookcrossing/BookcrossingServer/security/SecurityConfig.java

```diff
@@ -54,8 +39,4 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
                 .addFilterBefore(jwtFilter, UsernamePasswordAuthenticationFilter.class);
     }
 
-    @Bean
```

Comment:
Why was it necessary to move this bean to BookCrossingServerApplication.java?

Reply:
It was taken out due to the formation of a cyclical dependence

Reply:
I put the creation in a separate class